apps/hash-graph/lib/graph/src/shared/identifier/time/bound.rs

```diff
@@ -98,31 +89,31 @@ impl<A> TemporalTagged for LimitedTimeIntervalBound<A> {
 
     fn cast<T>(self) -> LimitedTimeIntervalBound<T> {
         match self {
-            Self::Included(limit) => LimitedTimeIntervalBound::Included(limit.cast()),
-            Self::Excluded(limit) => LimitedTimeIntervalBound::Excluded(limit.cast()),
+            Self::Inclusive(limit) => LimitedTimeIntervalBound::Inclusive(limit.cast()),
+            Self::Exclusive(limit) => LimitedTimeIntervalBound::Exclusive(limit.cast()),
         }
     }
 }
 
 impl<A> IntervalBound<Timestamp<A>> for LimitedTimeIntervalBound<A> {
     fn as_bound(&self) -> Bound<&Timestamp<A>> {
         match self {
-            Self::Included(limit) => Bound::Included(limit),
-            Self::Excluded(limit) => Bound::Excluded(limit),
+            Self::Inclusive(limit) => Bound::Included(limit),
+            Self::Exclusive(limit) => Bound::Excluded(limit),
```

Comment:
The `Bound`s are still called `Included` and `Excluded`, is that something we want to change in the `interval-ops` crate at some point?

Reply:
These are `use std::ops::Bound;`, I made the same mistake in a previous PR when asking @TimDiekmann 😁

Reply:
`std` uses `Included` and `Excluded`. This is the reason why I argued for `"included"` and `"excluded"` as well.

Reply:
> This is the reason why I argued for "included" and "excluded" as well.

Just for posterity, we mean in terms of the BP spec and the wider system. Instead we've chosen to have "inclusive" and "exclusive" across the system but (mostly) keep "included" and "excluded" within the Graph